src/dmd/backend/dwarfdbginf.d

```diff
@@ -1635,7 +1635,7 @@ static if (1)
         }
         abuf.writeByte(DW_AT_name);      abuf.writeByte(DW_FORM_string);
 
-        static if (DWARF_VERSION >= 4)
+        if (config.dwarf >= 4)
```

Comment:
Why support `> 4` when that is not supported?

Reply:
Differences between dwarf versions are mostly additive.  I don't think it makes sense to have `config.dwarf == 4` when the tag for `config.dwarf < 4` would be not correct for v5, v6, and so one.

Reply:
Put in an assert for `> 4` so that we know to check it when v5 exists. As for `< 4`, those versions aren't going to change so there won't be unanticipated code paths.